src/main/java/com/simondata/pouroversql/ExtractorRunner.java

```diff
@@ -67,6 +67,10 @@ private static Options getOptions() {
         options.addOption("fetchsize", "fetchsize", true, "Fetch size");
         options.addOption("timeout", "timeout", true, "Query Timeout in seconds");
         options.addOption("maxrows", "maxrows", true, "Maximum rows");
+        options.addOption("sftp", "sftp", false, "Connecting to SFTP");
```

Comment:
If it's a boolean, you can make that clearer to the user with a flag like `is-sftp` and the help can end with a question mark ;-)